specification/communication/Communication.JobRouter/tspconfig.yaml

```diff
@@ -46,7 +46,6 @@ options:
     generate-test: true
     generate-sample: true
   "@azure-tools/typespec-java":
-    emitter-output-dir: "{java-sdk-folder}/sdk/{service-directory-name}/azure-communication-jobrouter"
```

Comment:
@weidongxu-microsoft I dont see why they couldnt they just use this: 

```
emitter-output-dir: "{output-dir}/{service-dir}/azure-communication-jobrouter"
```

It looks like for some reason they're trying to manually parse and separate out the service-dir instead just using the variable directly.